apps/sports-helsinki/src/domain/search/combinedSearch/hooks/useFormValues.tsx

```diff
@@ -0,0 +1,80 @@
+import type { Option } from '@events-helsinki/components';
+import { useSearchTranslation } from '@events-helsinki/components';
+import React from 'react';
+import {
+  getTargetGroupOptions,
+  getSportsCategoryOptions,
+  sortExtendedCategoryOptions,
+  getAccessibilityShortcomingOptions,
+  sortOptionsAlphabetically,
+} from '../../eventSearch/utils';
+import { useCombinedSearchContext } from '../adapters/CombinedSearchContext';
+
+export const EMPTY_OPTION: Option = { text: '', value: '' };
+
+export function useFormValues() {
+  const { t } = useSearchTranslation();
+  const { formValues } = useCombinedSearchContext();
+  const [autosuggestInput, setAutosuggestInput] = React.useState(
+    formValues.text ?? ''
+  );
+  const [selectedSportsCategories, setSelectedSportsCategories] =
+    React.useState<string[]>(formValues.sportsCategories);
+  const [sportsCategoryInput, setSportsCategoryInput] = React.useState('');
+  const sportsCategories = getSportsCategoryOptions(t).sort(
+    sortExtendedCategoryOptions
+  );
+  const [selectedTargetGroups, setSelectedTargetGroups] = React.useState<
+    string[]
+  >(formValues.targetGroups);
+  const [targetGroupInput, setTargetGroupInput] = React.useState('');
+  const targetGroups = getTargetGroupOptions(t).sort(
+    sortExtendedCategoryOptions
+  );
```

Comment:
```suggestion
  const targetGroups = getTargetGroupOptions(t);
```

To not revert changes made with PR #520